### PR TITLE
AVTP pipeline improvements

### PR DESCRIPTION
**Pull request by MarcinMiklas**

### CODE REVIEW

**Comment by andrew-elder** [comment]
This looks ok to me. Anyone else?

File: lib/avtp_pipeline/platform/Linux/rawsock/pcap_rawsock.c [hunk 1]
```diff
@@ -66,9 +64,9 @@ void *pcapRawsockOpen(pcap_rawsock_t* rawsock, const char *ifname, bool rx_mode,
 	// Deal with frame size.
 	if (rawsock->base.frameSize == 0) {
 		// use interface MTU as max frames size, if none specified
-		rawsock->base.frameSize = rawsock->base.ifInfo.mtu;
+		rawsock->base.frameSize = rawsock->base.ifInfo.mtu + ETH_HLEN + VLAN_HLEN;
 	}
-	else if (rawsock->base.frameSize > rawsock->base.ifInfo.mtu) {
+	else if (rawsock->base.frameSize > rawsock->base.ifInfo.mtu + ETH_HLEN + VLAN_HLEN) {
```
Comment (davidcemin) [inline lib/avtp_pipeline/platform/Linux/rawsock/pcap_rawsock.c]: It does make sense to me to include the size of the vlan+ethernet headers here. Did we have a bug before on these lines?

**Comment by davidcemin** [comment]
Looks good to me as well.

File: lib/avtp_pipeline/platform/Linux/rawsock/pcap_rawsock.c (hunk 1 above)
Comment (MarcinMiklas) [inline lib/avtp_pipeline/platform/Linux/rawsock/pcap_rawsock.c]: Yes, we had bug here :)

**Pull request merged**